packages/api/AlvTimeWebApi/Startup.cs

```diff
@@ -45,6 +58,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (!env.IsDevelopment())
             {
                 app.UseHsts(); // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+            } else
+            {
+                app.UseCors(DevCorsPolicyName);

```

Comment:
Det står i PR'en at det legges til CORS policy for local development, men dette blir vel i prod? Eller skal det fjernes på et tidspunkt?